friendship/models.py

```diff
@@ -341,8 +331,8 @@ def add_friend(self, from_user, to_user, message=None):
     def remove_friend(self, from_user, to_user):
         """ Destroy a friendship relationship """
         try:
-            qs = Friend.objects.filter(Q(to_user=to_user, from_user=from_user) | Q(to_user=from_user, from_user=to_user))
-            distinct_qs = qs.distinct().all()
+            qs = Friend.objects.filter(to_user__in=[to_user, from_user], from_user__in=[from_user, to_user])
+            distinct_qs = qs.distinct()
```

Comment:
`.distinct()` is redundant too.

```diff
- distinct_qs = qs.distinct()
-
- if distinct_qs:
+ if qs:
                   friendship_removed.send(
-                     sender=distinct_qs[0], from_user=from_user, to_user=to_user
+                     sender=qs[0], from_user=from_user, to_user=to_user
                  )
```

Previously mentioned in https://github.com/revsys/django-friendship/pull/131#discussion_r621293829.

Reply:
👍